server/chat.ts

```diff
@@ -939,6 +939,13 @@ export class CommandContext extends MessageContext {
 
 		return true;
 	}
+	canPMLocked(user: User, lockedUser: User) {
+		const isPunishmentsAutolock = Punishments.userids.get(lockedUser.id)![3].includes('Autolocked for having punishments');
```

Comment:
```suggestion
		const isPunishmentsAutolock = Punishments.userids.get(lockedUser.id)?.[3].includes('Autolocked for having punishments');
```

Locked users are not necessarily in the Punishments table, from what I'm aware.

Reply:
huh